enigma-runtime-t/src/ocalls_t.rs

```diff
@@ -163,6 +185,14 @@ pub mod tests {
         assert_eq!(enc, ret);
     }
 
+    pub fn test_remove_delta(db_ptr: *const RawPointer) {
+        let contract_address = b"test_delta_removal".sha256();
+        let (start, end) = (0, 1);
+        let deltas = save_deltas(db_ptr, start, end, &contract_address);
+        let res = remove_delta(db_ptr, deltas.last().unwrap());
+        assert_eq!(res.unwrap(), ());
```

Comment:
Very minor. You may use `assert(res.is_ok())`

Reply:
changed. Thanks